### PR TITLE
Feature/home page

### PR DESCRIPTION
**Pull request by elishab-dev**

### CODE REVIEW

**Review by gretamas — APPROVED** [review]
Good job Lisa

File: index.html [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+    </header>
+    <main class="main-content">
+      <section class="home-page">
+        <img
+          src="./assets/img/we-serve-best-cakes.jpg"
+          alt="man-serving-cake"
+          class="home-page__img img-border"
+        />
+        <section class="home-page__text-button">
+          <h3 class="ff-saira fw700 f-s36 lh-15 bl pl25">
```
Comment (ilona-liashenko) [inline index.html]: I would suggest to use h2 tag here instead of h3 to use heading tags in a logical order

**Review by ilona-liashenko — APPROVED** [review]
Well-done!:)

File: index.html (hunk 1 above)
Comment (elishab-dev) [reply index.html]: ok

**Pull request merged**